jwql/website/apps/jwql/views.py

```diff
@@ -403,15 +407,28 @@ def dashboard(request):
     """
 
     template = 'dashboard.html'
-    output_dir = get_config()['outputs']
-    dashboard_components, dashboard_html = get_dashboard_components()
 
-    context = {'inst': '',
-               'outputs': output_dir,
-               'filesystem_html': os.path.join(output_dir, 'monitor_filesystem',
-                                               'filesystem_monitor.html'),
-               'dashboard_components': dashboard_components,
-               'dashboard_html': dashboard_html}
+    db = get_dashboard_components(request)
+    pie_graph = db.dashboard_instrument_pie_chart()
+    files_graph = db.dashboard_files_per_day()
+    filetype_bar = db.dashboard_filetype_bar_chart()
+    table_columns, table_values = db.dashboard_monitor_tracking()
+    grating_plot = db.dashboard_exposure_count_by_filter()
+    anomaly_plot = db.dashboard_anomaly_per_instrument()
+
+    p = layout([
```

Comment:
PEP8 frowns upon single-character variable names.  Perhaps this could be changed to `plot` instead?

Reply:
Looks like we forgot one. We should get you a bot that detects this and makes this review for every PR.